packages/x-charts/src/internals/plugins/featurePlugins/useChartClosestPoint/findClosestPoints.ts

```diff
@@ -43,10 +43,10 @@ export function findClosestPoints(
   }
 
   const pointX = originalXScale(
-    invertScale(xScale, svgPointX, (dataIndex) => seriesData[dataIndex].x),
+    invertScale(xScale, svgPointX, (dataIndex) => seriesData[dataIndex]?.x),
```

Comment:
This happens when you hover over the "margin" between chart and legend in the bar composition demo `/x/react-charts/bars/#composition`